qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -35,6 +35,10 @@ class FineAmplitudeCal(BaseCalibrationExperiment, FineAmplitude):
         :class:`FineAmplitudeCal` is a subclass of :class:`FineAmplitude`. In the calibration
         experiment the circuits that are run have a custom gate with the pulse schedule attached
         to it through the calibrations.
+
+    # section: see_also
+        qiskit_experiments.library.characterization.fine_amplitude.FineAmplitude
```

Comment:
Will the analysis ref section be inherited from parent here? If not this should also have 
```python
# section: analysis_ref
        :py:class:`FineAmplitudeAnalysis`
```

Reply:
This is automatically handled in parser.
https://github.com/Qiskit/qiskit-experiments/blob/6812ac230f024ca748a5de56750aa81f93808703/docs/_ext/custom_styles/formatter.py#L95-L101